api/Node.json

```diff
@@ -1603,10 +1618,12 @@
               "notes": "Only supported on <code>Element</code>."
             },
             "opera": {
-              "version_added": "7"
+              "version_added": "7",
+              "notes": "Before Opera 15, this feature was only supported on <code>Element</code>."
```

Comment:
I did some quick testing to confirm that `document.body.parentElement` works in Opera 12.16, but `Text` and `Attr` nodes didn't have `parentElement`. Those are the other node types where it most likely might have existing, so I think this was `Element` only.